pkgs/os-specific/linux/fsverity-utils/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, fetchgit
+, openssl
+, enableShared ? !stdenv.hostPlatform.isStatic
+, enableManpages ? false
```

Comment:
```suggestion
```
Why do we have that option at all? If they are in a different output you can disable installing man pages altogether if you don't want them.

Reply:
It wasnt about the size, it was about avoiding building pandoc on static/musl builds

I can make it default on

Reply:
I am not sure if we should add such flags. Couldn't we add pandoc to the packages that get build by hydra with a static/musl variant? That would improve this universally.

Reply:
That would be nice but this is a small override to maintain that makes it a lot easier to avoid it.

I've been building packages from scratch quite a bit with some of my experiments & some cross compiling etc and I think if a program can be built in 2 seconds without a couple manpages or 3 minutes+ with I'm happy to offer that override.

But again I'm happy to leave it default on

Reply:
> That would be nice but this is a small override to maintain that makes it a lot easier to avoid it.

I don't wanna see such overrides in more packages especially if people start to add them for more things they happen to find handy at some point :)

If they are not tested on updates such flags easily break over time and also increase the complexity. How about you do that locally in an overlay then we don't need to worry about it in nixpkgs?

Reply:
so to override you could do

```
fsverity-utils.overrideAttrs(oa: {
  nativeBuildInputs = [ ];
  installTargets = [ "install" ];
})
```

but if something then gets added to `nativeBuildInputs` or `installTargets` you'd have to copy it over by hand. Or maybe do some weird filter of the old attributes? :shrug:

---

worst case the non-default override doesn't work and people have to do the above themselves. and I don't mind maintaining it

Reply:
I still don't think this is the right approach to reduce build time on musl targets.

Reply:
```
λ rg 'enableManpages \?' --count
pkgs/development/compilers/llvm/11/clang/default.nix:1
pkgs/development/compilers/llvm/8/clang/default.nix:1
pkgs/development/compilers/llvm/12/llvm/default.nix:1
pkgs/development/compilers/llvm/12/clang/default.nix:1
pkgs/development/compilers/llvm/7/clang/default.nix:1
pkgs/development/compilers/llvm/6/llvm/default.nix:1
pkgs/development/compilers/llvm/12/lldb/default.nix:1
pkgs/development/compilers/llvm/10/llvm/default.nix:1
pkgs/development/compilers/llvm/13/llvm/default.nix:1
pkgs/development/compilers/llvm/13/clang/default.nix:1
pkgs/development/compilers/llvm/8/llvm/default.nix:1
pkgs/development/compilers/llvm/13/lldb/default.nix:1
pkgs/development/compilers/llvm/7/llvm/default.nix:1
pkgs/development/compilers/llvm/git/clang/default.nix:1
pkgs/development/compilers/llvm/git/lldb/default.nix:1
pkgs/development/compilers/llvm/10/lldb/default.nix:1
pkgs/development/compilers/llvm/14/lldb/default.nix:1
pkgs/development/compilers/llvm/11/llvm/default.nix:1
pkgs/development/compilers/llvm/rocm/llvm/default.nix:1
pkgs/development/compilers/llvm/14/llvm/default.nix:1
pkgs/development/compilers/llvm/git/llvm/default.nix:1
pkgs/development/compilers/llvm/10/clang/default.nix:1
pkgs/development/compilers/llvm/5/clang/default.nix:1
pkgs/development/compilers/llvm/5/llvm/default.nix:1
pkgs/development/compilers/llvm/14/clang/default.nix:1
pkgs/development/compilers/llvm/11/lldb/default.nix:1
pkgs/development/compilers/llvm/6/clang/default.nix:1
pkgs/development/compilers/llvm/9/clang/default.nix:1
pkgs/development/compilers/llvm/9/llvm/default.nix:1
λ rg 'enableDocs \?' --count
pkgs/development/libraries/opencv/4.x.nix:1
pkgs/development/libraries/pipewire/wireplumber.nix:1
pkgs/applications/science/electronics/xyce/default.nix:1
pkgs/applications/science/misc/openmvg/default.nix:1
pkgs/applications/science/math/singular/default.nix:1
pkgs/development/compilers/ghc/head.nix:1
pkgs/development/compilers/ghc/9.0.2.nix:1
pkgs/development/compilers/ghc/8.10.7.nix:1
pkgs/development/compilers/ghc/9.2.3.nix:1
pkgs/development/compilers/ghc/8.8.4.nix:1
```

I can make it default enabled on and have people override it off. Or I can have it turn off when built for something like musl or static

Reply:
oh, well. If you really want it...